app-web/src/templates/SourceMarkdown.module.css

```diff
@@ -11,7 +11,28 @@
     color: #494949;
     font-weight: 700;
     margin: 1em 0 0.6em 0;
-    font-family: 'Noto Sans'
+    font-family: 'Noto Sans', Verdana, sans-serif;
+    border-bottom: none;
+}
+
+.MarkdownBody h1{
```

Comment:
looks great just add a space between h1 and { --> `h1 {`
similar to the ones below